pkgi.cpp

```diff
@@ -141,6 +143,13 @@ static void pkgi_friendly_size(char* text, uint32_t textlen, int64_t size)
     }
 }
 
+void RefreshGames(const char* url, Mode set_mode ){
```

Comment:
As I said before, please respect the project's naming convention, and use clang-format on your file